### ISSUE TITLE
Add how to place a pull request on the Project Board in CONTRIBUTING.md

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#2641]
### Overview
As developers, we need to make sure our guide on how to contribute is up to date. In this issue, we will include a section on how to add a new pull request to the Project Board.

### Action Items
- [x] Currently in [CONTRIBUTING.md](https://github.com/hackforla/website/blob/e33b5d42e8c048255b2448ca5567f511c85dc1c2/CONTRIBUTING.md), there is a section 3.1.b.[iv. Complete pull request (4): Include images (if available)](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#iv-complete-pull-request-4-include-images-if-available) and a section 3.1.b.[v. After pull request is submitted/merged](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#v-after-pull-request-is-submittedmerged). We will be adding a section between these two sections.
- [x] For the current section 3.1.b.[v. After pull request is submitted/merged](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#v-after-pull-request-is-submittedmerged), renumber it so the `v` is `vi`.
  - [x] Update the table of contents to reflect the change in numbering.
    - [x] Also, update the link for the section in the table of contents. 
- [x] We are now going to add a new section 3.1.b.v titled, "How to add a pull request to the project board" with instructions on how to add a pull request to the project board.
  - [x] After 3.1.b.iv in [CONTRIBUTING.md](https://github.com/hackforla/website/blob/e33b5d42e8c048255b2448ca5567f511c85dc1c2/CONTRIBUTING.md), add the new section 3.1.b.v titled, "How to add your pull request to the project board".
  - [x] Please include the following at the beginning of your instructions:
    > We are in the process of automating newly created pull requests being added to the Project Board. However, it still should be manually checked to make sure the pull request has been placed on the Project Board in case automation breaks.  
  - [x] Then, write instructions for how to add the pull request to the Project Board by clicking the gear icon on the right under Projects and select Project Board
    <img width="236" alt="Project-selection" src="https://user-images.githubusercontent.com/31293603/147790802-bb60f20e-c550-40b8-a148-dd8d21e927ad.png">
  - [x] You may want to include pictures with your instructions
- [x] Update the Table of Contents with a link to the new section. Follow the current format of the Table of Contents. 
- [x] Commit and push changes to your branch on your forked repository
- [x] In CONTRIBUTING.md on your remote branch, check the changes you made.
  - [x] Make sure the links in the Table of Contents work as expected.

#### Merge Team or Technical Leads
Once the pull request associated with this issue is approved and merged, please update and edit issue #2670 by
- [x] Checking off the dependency for this issue
- [x] Under the Resources/Instructions, adding a link to the newly written section on how to place a pull request
- [ ] If both dependencies are checked off, please move issue #2670 to the New Issue Approval column and remove the Dependency label

### Resources/Instructions
- [CONTRIBUTING.md](https://github.com/hackforla/website/blob/e33b5d42e8c048255b2448ca5567f511c85dc1c2/CONTRIBUTING.md)

**Comment by lindseyindev** [comment]
One thing I would recommend specifying in the instructions is the step to select the website project on the pull request

<details>
<img src="https://gyazo.com/fe19d5ca303074fabd0b0922c2821260.png"/>
</details>

**Comment by SAUMILDHANKAR** [comment]
@JessicaLucindaCheng: We have issue - #2395 in prioritized backlog for automatically moving PRs into the project board. So, this issue might not be required (depends on which one gets merged first).

**Comment by JessicaLucindaCheng** [comment]
> One thing I would recommend specifying in the instructions is the step to select the website project on the pull request
> 
> ![](https://camo.githubusercontent.com/ac7591c8419cfbe80cc7e2d5bef4c97f47c4ce7b888fa741d3317a92197aba35/68747470733a2f2f6779617a6f2e636f6d2f66653139643563613330333037346661626430623039323263323832313236302e706e67)

@lindseyindev Good suggestion! I updated my original post to try to include your recommendation.

**Comment by JessicaLucindaCheng** [comment]
> @JessicaLucindaCheng: We have issue - #2395 in prioritized backlog for automatically moving PRs into the project board. So, this issue might not be required (depends on which one gets merged first).

@SAUMILDHANKAR Since it will be fixed in issue #2395, should we close this issue then?

**Comment by SAUMILDHANKAR** [comment]
@JessicaLucindaCheng It might take a while before that issue is worked upon and there might be a lot of PRs which might benefit from this in the meantime. Also, it is a good practice to double check if it has been placed in the project board in case the automation fails for some reason. So, good idea to keep this issue. I will add ready for milestone.

**Comment by ExperimentsInHonesty** [comment]
What do you think about putting a note on the other issue #2395 to revert this pr once the other is done?

**Comment by ExperimentsInHonesty** [comment]
@JessicaLucindaCheng is going to revise to indicate in the contributing file that its being automated and that they should still check manually to confirm in case of automation breakage. 

Please put the ready for milestone label back on this when you are done.

**Comment by JessicaLucindaCheng** [comment]
Revision ETA: Tue, Jan 4, 2022

**Comment by SAUMILDHANKAR** [comment]
@JessicaLucindaCheng: Thanks for the revision. Just added a small note for future changes.

**Comment by JessicaLucindaCheng** [comment]
@SAUMILDHANKAR Thanks for looking at my issue write up again! Instead of adding that line to the Wiki, would it be better to create an issue dependent on this issue and the GHA one too? However, I'm not sure if an issue can have more than one dependency. My reason is creating a note in the contributing.md to do something later may be missed or forgotten later. Let me know what you think.

**Comment by JessicaLucindaCheng** [comment]
Spoke with @SAUMILDHANKAR and he agreed we should create a new issue dependent on this issue and dependent on the Issue https://github.com/hackforla/website/issues/2395 for automating putting PRs on the board. I will create this new issue.

**Comment by JessicaLucindaCheng** [comment]
@ExperimentsInHonesty I revised the issue as requested.

**Comment by JessicaLucindaCheng** [comment]
@JessicaLucindaCheng After speaking with @Providence-o, what section to add to the contributing.md. Jessica will make changes to clarify that in this issue.

**Comment by JessicaLucindaCheng** [comment]
> @JessicaLucindaCheng After speaking with @Providence-o, what section to add to the contributing.md. Jessica will make changes to clarify that in this issue.

I made the clarifications as requested.

**Comment by JessicaLucindaCheng** [comment]
@Providence-o I made the clarifications as requested.

> @JessicaLucindaCheng After speaking with @Providence-o, what section to add to the contributing.md. Jessica will make changes to clarify that in this issue.

**Comment by kathrynsilvaconway** [comment]
ETA: Before 6pm on 4/02/2022
Availability: 9am-12pm on Sat, 4/02. 
*Likely it will be done by tomorrow night, but Sat is the time I can commit to.

**Comment by kathrynsilvaconway** [comment]
Test adding this image to my issue
![add_pr_to_pb](https://user-images.githubusercontent.com/81049661/161181526-23ae511c-e991-4cc6-a0a4-d8db19dd69ca.png)
:
